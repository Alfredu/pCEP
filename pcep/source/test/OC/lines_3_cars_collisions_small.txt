detect_obj_loc+avg.dolce
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 8.88029999999999 float t 14770104513000
1 obj_loc int obj_id 2 string type L float loc_x 8.61219999999999 float loc_y 0.5803 float t 14770104513000
1 obj_loc int obj_id 3 string type L float loc_x 18.6122000000002 float loc_y 0.3803 float t 14770104513000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 8.98029999999999 float t 14770104514000
1 obj_loc int obj_id 2 string type L float loc_x 8.71219999999999 float loc_y 0.5803 float t 14770104514000
1 obj_loc int obj_id 3 string type L float loc_x 18.4122000000002 float loc_y 0.3803 float t 14770104514000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.08029999999999 float t 14770104515000
1 obj_loc int obj_id 2 string type L float loc_x 8.81219999999999 float loc_y 0.5803 float t 14770104515000
1 obj_loc int obj_id 3 string type L float loc_x 18.2122000000002 float loc_y 0.3803 float t 14770104515000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.18029999999998 float t 14770104516000
1 obj_loc int obj_id 2 string type L float loc_x 8.91219999999999 float loc_y 0.5803 float t 14770104516000
1 obj_loc int obj_id 3 string type L float loc_x 18.0122000000002 float loc_y 0.3803 float t 14770104516000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.28029999999998 float t 14770104517000
1 obj_loc int obj_id 2 string type L float loc_x 9.01219999999999 float loc_y 0.5803 float t 14770104517000
1 obj_loc int obj_id 3 string type L float loc_x 17.8122000000002 float loc_y 0.3803 float t 14770104517000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.38029999999998 float t 14770104518000
1 obj_loc int obj_id 2 string type L float loc_x 9.11219999999999 float loc_y 0.5803 float t 14770104518000
1 obj_loc int obj_id 3 string type L float loc_x 17.6122000000002 float loc_y 0.3803 float t 14770104518000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.48029999999998 float t 14770104519000
1 obj_loc int obj_id 2 string type L float loc_x 9.21219999999999 float loc_y 0.5803 float t 14770104519000
1 obj_loc int obj_id 3 string type L float loc_x 17.4122000000002 float loc_y 0.3803 float t 14770104519000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.58029999999998 float t 14770104520000
1 obj_loc int obj_id 2 string type L float loc_x 9.31219999999998 float loc_y 0.5803 float t 14770104520000
1 obj_loc int obj_id 3 string type L float loc_x 17.2122000000002 float loc_y 0.3803 float t 14770104520000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.68029999999998 float t 14770104521000
1 obj_loc int obj_id 2 string type L float loc_x 9.41219999999998 float loc_y 0.5803 float t 14770104521000
1 obj_loc int obj_id 3 string type L float loc_x 17.0122000000002 float loc_y 0.3803 float t 14770104521000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.78029999999998 float t 14770104522000
1 obj_loc int obj_id 2 string type L float loc_x 9.51219999999998 float loc_y 0.5803 float t 14770104522000
1 obj_loc int obj_id 3 string type L float loc_x 16.8122000000002 float loc_y 0.3803 float t 14770104522000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.88029999999998 float t 14770104523000
1 obj_loc int obj_id 2 string type L float loc_x 9.61219999999998 float loc_y 0.5803 float t 14770104523000
1 obj_loc int obj_id 3 string type L float loc_x 16.6122000000002 float loc_y 0.3803 float t 14770104523000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.98029999999998 float t 14770104524000
1 obj_loc int obj_id 2 string type L float loc_x 9.71219999999998 float loc_y 0.5803 float t 14770104524000
1 obj_loc int obj_id 3 string type L float loc_x 16.4122000000002 float loc_y 0.3803 float t 14770104524000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 10.0803 float t 14770104525000
1 obj_loc int obj_id 2 string type L float loc_x 9.81219999999998 float loc_y 0.5803 float t 14770104525000
1 obj_loc int obj_id 3 string type L float loc_x 16.2122000000002 float loc_y 0.3803 float t 14770104525000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 10.1803 float t 14770104526000
1 obj_loc int obj_id 2 string type L float loc_x 9.91219999999998 float loc_y 0.5803 float t 14770104526000
1 obj_loc int obj_id 3 string type L float loc_x 16.0122000000002 float loc_y 0.3803 float t 14770104526000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 10.2803 float t 14770104527000
1 obj_loc int obj_id 2 string type L float loc_x 10.0122 float loc_y 0.5803 float t 14770104527000
1 obj_loc int obj_id 3 string type L float loc_x 15.8122000000002 float loc_y 0.3803 float t 14770104527000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 10.3803 float t 14770104528000
1 obj_loc int obj_id 2 string type L float loc_x 10.1122 float loc_y 0.5803 float t 14770104528000
1 obj_loc int obj_id 3 string type L float loc_x 15.6122000000002 float loc_y 0.3803 float t 14770104528000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 10.4803 float t 14770104529000
1 obj_loc int obj_id 2 string type L float loc_x 10.2122 float loc_y 0.5803 float t 14770104529000
1 obj_loc int obj_id 3 string type L float loc_x 15.4122000000002 float loc_y 0.3803 float t 14770104529000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 10.5803 float t 14770104530000
1 obj_loc int obj_id 2 string type L float loc_x 10.3122 float loc_y 0.5803 float t 14770104530000
1 obj_loc int obj_id 3 string type L float loc_x 15.2122000000002 float loc_y 0.3803 float t 14770104530000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 10.6803 float t 14770104531000
1 obj_loc int obj_id 2 string type L float loc_x 10.4122 float loc_y 0.5803 float t 14770104531000
1 obj_loc int obj_id 3 string type L float loc_x 15.0122000000002 float loc_y 0.3803 float t 14770104531000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 10.7803 float t 14770104532000
1 obj_loc int obj_id 2 string type L float loc_x 10.5122 float loc_y 0.5803 float t 14770104532000
1 obj_loc int obj_id 3 string type L float loc_x 14.8122000000002 float loc_y 0.3803 float t 14770104532000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 10.8803 float t 14770104533000
1 obj_loc int obj_id 2 string type L float loc_x 10.6122 float loc_y 0.5803 float t 14770104533000
1 obj_loc int obj_id 3 string type L float loc_x 14.6122000000002 float loc_y 0.3803 float t 14770104533000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 10.9803 float t 14770104534000
1 obj_loc int obj_id 2 string type L float loc_x 10.7122 float loc_y 0.5803 float t 14770104534000
1 obj_loc int obj_id 3 string type L float loc_x 14.4122000000002 float loc_y 0.3803 float t 14770104534000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 11.0803 float t 14770104535000
1 obj_loc int obj_id 2 string type L float loc_x 10.8122 float loc_y 0.5803 float t 14770104535000
1 obj_loc int obj_id 3 string type L float loc_x 14.2122000000002 float loc_y 0.3803 float t 14770104535000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 11.1803 float t 14770104536000
1 obj_loc int obj_id 2 string type L float loc_x 10.9122 float loc_y 0.5803 float t 14770104536000
1 obj_loc int obj_id 3 string type L float loc_x 14.0122000000002 float loc_y 0.3803 float t 14770104536000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 11.2803 float t 14770104537000
1 obj_loc int obj_id 2 string type L float loc_x 11.0122 float loc_y 0.5803 float t 14770104537000
1 obj_loc int obj_id 3 string type L float loc_x 13.8122000000002 float loc_y 0.3803 float t 14770104537000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 11.3803 float t 14770104538000
1 obj_loc int obj_id 2 string type L float loc_x 11.1122 float loc_y 0.5803 float t 14770104538000
1 obj_loc int obj_id 3 string type L float loc_x 13.6122000000003 float loc_y 0.3803 float t 14770104538000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 11.4803 float t 14770104539000
1 obj_loc int obj_id 2 string type L float loc_x 11.2122 float loc_y 0.5803 float t 14770104539000
1 obj_loc int obj_id 3 string type L float loc_x 13.4122000000003 float loc_y 0.3803 float t 14770104539000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 11.5803 float t 14770104540000
1 obj_loc int obj_id 2 string type L float loc_x 11.3122 float loc_y 0.5803 float t 14770104540000
1 obj_loc int obj_id 3 string type L float loc_x 13.2122000000003 float loc_y 0.3803 float t 14770104540000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 11.6803 float t 14770104541000
1 obj_loc int obj_id 2 string type L float loc_x 11.4122 float loc_y 0.5803 float t 14770104541000
1 obj_loc int obj_id 3 string type L float loc_x 13.0122000000003 float loc_y 0.3803 float t 14770104541000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 11.7803 float t 14770104542000
1 obj_loc int obj_id 2 string type L float loc_x 11.5122 float loc_y 0.5803 float t 14770104542000
1 obj_loc int obj_id 3 string type L float loc_x 12.8122000000003 float loc_y 0.3803 float t 14770104542000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 11.8803 float t 14770104543000
1 obj_loc int obj_id 2 string type L float loc_x 11.6122 float loc_y 0.5803 float t 14770104543000
1 obj_loc int obj_id 3 string type L float loc_x 12.6122000000003 float loc_y 0.3803 float t 14770104543000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 11.9803 float t 14770104544000
1 obj_loc int obj_id 2 string type L float loc_x 11.7122 float loc_y 0.5803 float t 14770104544000
1 obj_loc int obj_id 3 string type L float loc_x 12.4122000000003 float loc_y 0.3803 float t 14770104544000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 12.0803 float t 14770104545000
1 obj_loc int obj_id 2 string type L float loc_x 11.8122 float loc_y 0.5803 float t 14770104545000
1 obj_loc int obj_id 3 string type L float loc_x 12.2122000000003 float loc_y 0.3803 float t 14770104545000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 12.1803 float t 14770104546000