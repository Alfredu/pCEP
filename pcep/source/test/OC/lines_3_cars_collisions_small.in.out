2 collision_with_moving_object float t 14770104172544.000000 float loc_y 9.680300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 9.680300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 9.780300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 9.780300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 9.880300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 9.880300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 9.980300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 9.980300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 10.080300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 10.080300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 10.180300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 10.180300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 10.280300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 10.280300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 10.380300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 10.380300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 10.480300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 10.480300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 10.580300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 10.580300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 10.680300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 10.680300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 10.780300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 10.780300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 10.880300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 10.880300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 10.980300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 10.980300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 11.080300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 11.080300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 11.180300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 11.180300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 11.280300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 11.280300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 11.380300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 11.380300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 11.480300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 11.480300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 11.580300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 11.580300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 11.680300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 11.680300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 11.780300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 11.780300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 11.880300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 11.880300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 11.980300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 11.980300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 12.080300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 12.080300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 12.180300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 12.180300
7 Shutdown int Result 0 
